### DISCUSSION TITLE
Possible to force ARM?

### DISCUSSION THREAD

**Discussion by ckcr4lyf** [waydroid/waydroid#1031] [Q&A] [answered]
I am trying to run an app, which has some compiled libs (JNI - `.so` files). But these are only present for ARM.

As a result, when it runs on waydroid (even with NDK / libhoudini), since the default app _CAN_ run on x86, it loads, without the ARM libs. Can I trick the app into thinking it is on ARM, so it will load those libs as well? 

Thanks.

**Comment by ckcr4lyf** [comment, marked as answer]
FYI: Seems it is possible, you can do it like this:

```
pm install --abi arm64-v8a [APK_PATH]
```

Thanks to ayasa520 - https://github.com/casualsnek/waydroid_script/issues/114#issuecomment-1661691568

**Reply by SerJaimeLannister** [reply]
hey what is this pm?
I really am not able to understand it.

**Reply by ckcr4lyf** [reply]
Command to install apps via `adb`
